src/Interfaces/ProtectorsConfigInterface.ts

```diff
@@ -3,5 +3,6 @@ import { LivestockConfigInterface } from './LivestockConfigInterface';
 
 export interface ProtectorsConfigInterface
   extends LivestockConfigInterface {
-  chasesAway: AnimalNames;
+  chasesAway: AnimalNames | undefined;
+  exclamation: string | undefined;
```

Comment:
Is it necessary to declare exclamation or chasesAway as undefined? I think about displaying exclamation in alert component, when herd will be protected. And i think that protector have to chase something away, otherwise it will lost it's purpose.

Reply:
Yes, I agree. Will be modified.